CheatSheet.md

```diff
@@ -362,8 +362,8 @@ NOTE: For the correct code convention of using postfix or not, read [this](https
 
     val r: Range = 1 until 5 // 1, 2, 3, 4
     val s: Range = 1 to 5    // 1, 2, 3, 4, 5
-    1 to 10 by 3  // 1, 4, 7, 10
-    6 to 1 by -2  // 6, 4, 2
+    1 to 10 by 3 toList // 1, 4, 7, 10
+    6 to 1 by -2 toList // 6, 4, 2
```

Comment:
Thank you for your suggestion! I think this does not follow the recommended code style, and I think it would be better to illustrate how to convert a range to a list in separate lines than these lines, who already have another purpose.

Therefore, I suggest adding another line like the following instead of changing these two lines:

~~~ scala
val xs = (1 to 10).toList
~~~

Reply:
hi @julienrf, I made a small fix following your suggestion, many thanks